game.cpp

```diff
@@ -0,0 +1,111 @@
+#include "game.h"
+#include <cmath>
+
+Game::Game(int fieldSize){
+    gameField.createField(fieldSize);
+    winStatus = false;
+    loseStatus = false;
+}
+
+void Game::updateWinStatus(){
+    std::vector<std::vector<Cell>>& currentField = gameField.getField();
+    for(int i = 0; i < gameField.getSize(); i++){
+        for(int j = 0; j < gameField.getSize(); j++){
+            if(currentField[i][j].haveWinValue()){
+                winStatus = true;
+                return;
+            }
+        }
+    }
+    winStatus = false;
+}
+
+void Game::updateLoseStatus(){
+    loseStatus = (!gameField.getMovableStatus());
+}
+
+void Game::moveByMouse(std::vector<std::vector<int>> mouseCoords){
```

Comment:
Check older comments, should not be any mouse related logic here

Reply:
done